targets/TARGET_STM/TARGET_STM32L4/TARGET_MTS_DRAGONFLY_L471QG/mbed_lib.json

```diff
@@ -0,0 +1,10 @@
+{
+	"name": "target",
+	    "config": {
+	"target_overrides": {
+        "MTS_DRAGONFLY_L471QG": {
```

Comment:
can this be set in the targets.json file (config can be set there as well) ?

Reply:
actually i had a question about how best to do that. if i wish to set a default boot-loader, should i do it in the target.json file?

Reply:
yes, that would be good to have it there (all in one place)